packages/suite-desktop/src-electron/mocks/electron-updater.ts

```diff
@@ -92,7 +113,7 @@ const autoUpdater = {
                     total: 1024 * 1024 * 10,
                     transferred: 1024 * 1024 * (i / 2),
                 });
-            }, 500);
+            }, 300);
```

Comment:
This is just for the mock, but makes the fake download faster.

Reply:
Aha, it would be really nice to put all these small tangential improvements to a separate commit. It would make the review much easier as the reviewer could focus on the important parts.

Reply:
Good point, I'll try to keep this in mind next time.

Reply:
> I'll try to keep this in mind next time.

Please. .)

See https://en.wikipedia.org/wiki/Atomic_commit#Atomic_commit_convention for motivation for doing this.